x-pack/plugins/apm/public/components/alerting/service_overview_alerts/alerts_table_flyout_body.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import type { AlertsTableFlyoutBaseProps } from '@kbn/triggers-actions-ui-plugin/public';
```

Comment:
FYI, I'm working with the Response ops team in order to consume the Flyout from table. Ideally we'll remove this and the `get_render_cell_value`